mvp-spec/audit-log-requirements.md

```diff
@@ -315,9 +315,10 @@ different technical objects including the "imp" object (for impression). It can
 be a location for a Transmission request for each placement;
 * a Bid response can also be extended (field "ext") in different objects. It 
 can be a place for Transmission responses.
-* It is acceptable in OpenRTB to send an empty response for a no-bid. It must 
-be considered as a Receiver failure of the Prebid SSO Transmission if the
-Transmission relies on the Bid request/response..
+* It is acceptable in OpenRTB to send an empty response for a no-bid. In this
+case, the Transmissions must end with a failing status. This should not impact
```

Comment:
I thought in this case we will accept not having transmission response, so why should this transmission end with a failing status? And where exactly is this failure status reported since there is no transmission response?

Reply:
You pointed out a dissonance, indeed. On one side, we accept the no bid case to be more aligned with OpenRTB. On the other side, we consider it as an error to be more aligned with the Model Terms.

From the Model Terms:
> On receipt of the SWAN Data, the Receiver shall Sign the Transaction to confirm receipt of the SWAN Data. The Receiver shall not use any SWAN Data where the data relating to a User’s Preference as required under clause 5(9)(ii) is missing from the SWAN Data or contains a null value.
>
> If the Receiver fails to Sign the Transaction but provides a response to that Transaction, then the Sender must Sign to confirm that it sent the SWAN Data to that Receiver as soon as it becomes aware of the Receiver’s failure.
>
> If the Receiver fails to Sign the Transaction but does not provide a response to that Transaction, then the Sender should Sign to confirm that it sent the SWAN Data to that Receiver as soon as it becomes aware of the Receiver’s failure.


To solve this issue, I suggest a new status "NO_BID". So we would have "SUCCESS", "FAIL" and "NO_BID".

Reply:
I have pushed an update accordingly to this suggestion. Don't hesitate to share other idea to solve this issue if it doesn't fit you.